mspec/lib/mspec/helpers.rb

```diff
@@ -3,7 +3,6 @@
 require 'mspec/helpers/datetime'
 require 'mspec/helpers/ducktype'
 require 'mspec/helpers/encode'
-require 'mspec/helpers/enumerator_class'
```

Comment:
How we can sync this and https://github.com/rubinius/mspec ? Should I update https://github.com/rubinius/mspec first? 🤔

Reply:
No one uses rubinius/mspec so you don't need to update it first. At some point, we can copy the changes over, but I don't think it is a big deal.

Reply:
Thanks for the clarify! 🙇 📝